Source/WebKit/UIProcess/Automation/mac/WebAutomationSessionMac.mm

```diff
@@ -789,6 +799,29 @@ static BOOL characterIsProducedUsingShift(NSString *characters)
 }
 #endif // ENABLE(WEBDRIVER_KEYBOARD_INTERACTIONS)
 
+#if ENABLE(WEBDRIVER_WHEEL_INTERACTIONS)
+
+void WebAutomationSession::platformSimulateWheelInteraction(WebPageProxy& page, const WebCore::IntPoint& locationInViewport, const WebCore::IntSize& delta)
+{
+    static constexpr auto scrollWheelCount = 2;
+    auto cgScrollEvent = adoptCF(CGEventCreateScrollWheelEvent(nullptr, kCGScrollEventUnitPixel, scrollWheelCount, -delta.height(), -delta.width()));
+
+    auto locationInWindow = viewportLocationToWindowLocation(locationInViewport, page);
+    NSWindow *window = page.platformWindow();
+
+    // Set the CGEvent location in flipped coords relative to the first screen, which compensates for the behavior of
+    // +[NSEvent eventWithCGEvent:] when the event has no associated window. See <rdar://problem/17180591>.
+    auto locationOnScreen = [window convertPointToScreen:locationInWindow];
+    locationOnScreen = CGPointMake(locationOnScreen.x, NSScreen.screens.firstObject.frame.size.height - locationOnScreen.y);
```

Comment:
Should we use something like `window.screen.frame.size.height` instead?

Reply:
I'll need to do some more testing here to determine if that works. I realize now that both my monitors are the exact same resolution, so this will always be right on my machine currently 😅

Reply:
I just double checked with two different screen resolutions and some silly screen arrangements and this does in fact need to be the height of the **first** screen, not the screen containing our eventual target window.